clang/test/SemaSYCL/redeclaration-attribute-propagation.cpp

```diff
@@ -77,7 +72,7 @@ int main() {
 
 #else
     h.single_task<class test_kernel2>(
-        []() { func2(); }); // expected-error {{conflicting attributes applied to a SYCL kernel or SYCL_EXTERNAL function}}
+        []() { func2(); });
```

Comment:
We do not give this diagnostic anymore because
SYCLIntelMaxWorkGroupSizeAttr *Sema::MergeSYCLIntelMaxWorkGroupSizeAttr(
    Decl *D, const SYCLIntelMaxWorkGroupSizeAttr &A) { -- This handles redeclaration of SYCLIntelMaxWorkGroupSizeAttr now instead of checkDimensionsAndSetDiagnostics<SYCLIntelMaxWorkGroupSizeAttr>(*this, New, Old) and PropagateAndDiagnoseDeviceAttr() for Propagating the attributes from redeclarations to SYCL kernel (https://github.com/intel/llvm/pull/2063)